packages/api/src/controllers/broadcaster.js

```diff
@@ -59,4 +59,23 @@ app.get('/status', authMiddleware({}), async (req, res, next) => {
   res.json(statuses)
 })
 
+async function discoveryAuthWebhookHandler(req, res) {
+  let responseObj = {}
+  let broadcasters = []
+  try {
+    broadcasters = await req.getBroadcasters(req)
+  } catch (err) {
+    return res.status(400).send({message: err.message})
+  }
+  for (let i = 0; i < broadcasters.length; i++) {
+    if (broadcasters[i].eth.address == req.body.id) {
+      responseObj["priceInfo"] = broadcasters[i].eth?.priceInfo
```

Comment:
priceInfo is used as the field name here, but the go-livepeer PR expects [price](https://github.com/livepeer/go-livepeer/pull/1736/files#diff-7157f6a8ee36dc4ada0f69ca25c9b79cbbae048dba0b4dde96270ad400d900eeR326).

Reply:
fixed in the go-livepeer PR